### ISSUE TITLE
Invisible fonts on microsoft.com

### ISSUE THREAD

**Issue by annulen** [annulen/webkit#45]
https://products.office.com/EN-US/whats-new-office?Wt.mc_id=oan_EN-US_mscom_prog_office2016launchwhatsnew#

```css
@font-face {
	font-family:'wf_segoe-ui_light';
	src:url("//i.s-microsoft.com/fonts/segoe-ui/west-european/light/latest.eot");
	src:local("Segoe UI Light"),local("Segoe WP Light"),url("//i.s-microsoft.com/fonts/segoe-ui/west-european/light/latest.eot?#iefix") format("embedded-opentype"),url("//i.s-microsoft.com/fonts/segoe-ui/west-european/light/latest.woff") format("woff"),url("//i.s-microsoft.com/fonts/segoe-ui/west-european/light/latest.ttf") format("truetype");
	font-weight:normal;
	font-style:normal;
	-webkit-font-smoothing:antialiased
}
```

**Comment by annulen** [comment]
It uses WOFF fonts (note that we support WOFF in other places)

**Comment by annulen** [comment]
Similar bug can be seen on https://davidwalsh.name/demo/html5-context-menu.php

**Comment by annulen** [comment]
We need to implement WebCore::FontCascade::getGlyphsAndAdvancesForComplexText

**Comment by annulen** [comment]
Fixed in 76c769cb44da888a473e8ba5fc78ae88a01b833f
